api/src/main/java/io/opencensus/trace/TraceId.java

```diff
@@ -211,12 +213,13 @@ public boolean equals(@Nullable Object obj) {
     }
 
     TraceId that = (TraceId) obj;
-    return Arrays.equals(bytes, that.bytes);
+    return idHi == that.idHi && idLo == that.idLo;
   }
 
   @Override
   public int hashCode() {
-    return Arrays.hashCode(bytes);
+    // TODO: Use Objects.hashCode when switch to java17.
+    return Arrays.hashCode(new Long[] {idHi, idLo});
```

Comment:
is it worth to avoid creating intermediate Long[]?

Reply:
done